mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/AbstractProfileAssociationDao.java

```diff
@@ -25,6 +38,9 @@
 public abstract class AbstractProfileAssociationDao<M, D> implements ProfileAssociationDao<M, D> {
   protected static final String ID_FIELD = "'id'";
   private static final Logger logger = LoggerFactory.getLogger(AbstractProfileAssociationDao.class);
+  private static final String RETRIEVES_DETAILS_SQL = "Select * from associations_view Where master_id='%s'";
```

Comment:
Please specify the exact fields instead of star selection (select *)

Reply:
Done